### PR TITLE
helm: fixes cmd arguments

### PR DESCRIPTION
**Pull request by mheers**
corrects metrics-bind-address, leader-elect and cert-manager-enabled

| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?
Fixes command line arguments (metrics-bind-address, leader-elect and cert-manager-enabled) for starting nifikop.

### Why?
The helm charts `deployment.yaml` uses incompatible command line arguments for starting nifikop.

### Checklist

- [ ] Implementation tested
- [ ] Error handling code meets the [guideline](docs/error-handling-guide.md)
- [ ] Logging code meets the guideline
- [ ] User guide and development docs updated (if needed)
- [ ] Append changelog with changes

### CODE REVIEW

File: helm/nifikop/templates/deployment.yaml [hunk 1]
```diff
@@ -79,10 +79,10 @@ spec:
         - command:
             - /manager
           args:
-            - metrics-bind-address={{ .Values.metrics.port }}
-            - --leader-elect
+            - -metrics-bind-address=:{{ .Values.metrics.port }}
```
Comment (michael81877) [inline helm/nifikop/templates/deployment.yaml:82]: I know the defaults for these port properties have colons in them, but are we sure that the port configuration works with and without the colon?

Comment (mheers) [reply helm/nifikop/templates/deployment.yaml:82]: `.Values.metrics.port` has to be a number as we use `containerPort: {{ .Values.metrics.port }}` which expects a number.

When having no colon in `metrics-bind-address` the operator fails with `{"level":"error","time":"2022-10-25T11:59:25.008+0300","caller":"nifikop/main.go:88","msg":"unable to start manager","error":"error listening on 8089: listen tcp: address 8089: missing port in address","stacktrace":"main.main\n\t/home/marcel/workspace/sick/git/github/nifikop/main.go:88\nruntime.main\n\t/usr/lib/go-1.19/src/runtime/proc.go:250"}`

**Comment by michael81877** [comment]
LGTM apart from one minor comment

File: helm/nifikop/templates/deployment.yaml (hunk 1 above)
Comment (michael81877) [reply helm/nifikop/templates/deployment.yaml:82]: Thanks for the clarification

**Review by michael81877 — APPROVED** [review]

**Review by erdrix — APPROVED** [review]

**Pull request merged**